lib/internal/async_hooks.js

```diff
@@ -270,6 +272,11 @@ function getDefaultTriggerAsyncId() {
 }
 
 
+function clearDefaultTriggerAsyncId() {
```

Comment:
nit: It might be worth moving the copy of this operation in `env-inl.h` into a standalone `create_default_trigger_async_id` function so that we can attach a comment in both places to keep the copies in sync.

Aside: I've wanted to cleanup the magical -1 and 0 hard-coded constants for async id, but that should probably happen in a separate PR.

Reply:
I don't know if it makes sense to move it off into a function in C++ since it seems to only be done from the constructor. Perhaps a constant makes the most sense.

Reply:
Could you add a comment that says that this should never be used except for exception handling?